picard/script.py

```diff
@@ -372,27 +421,6 @@ def _compute_logic(operation, *args):
     return operation(args)
 
 
-def _get_multi_values(parser, multi, separator):
-    if isinstance(separator, ScriptExpression):
-        separator = separator.eval(parser)
-
-    if separator == MULTI_VALUED_JOINER:
```

Comment:
@phw Is this correct? I mean if separator is evaluated to `'; '` (value of `MULTI_VALUED_JOINER`), do we still want go through this branch?

Reply:
The intention seems to be yes. But to be honest I don't really get the code here. Why do we have special handling if the separator is the default?

Reply:
I don't get the code either, hence the question. I'll have a deeper look after the release and before we merge this PR.